### PR TITLE
Add recipes required for experimental features

### PR DESCRIPTION
**Pull request by psaavedra**

### CODE REVIEW

**Comment by philn** [comment]
Can you make a new option for WebRTC? If experimental features are enabled I would prefer to have WebRTC disabled unless a dedicated new PACKAGECONFIG option is enabled. Then you could make it depend on libvpx too.

**Comment by psaavedra** [comment]
> Can you make a new option for WebRTC? If experimental features are enabled I would prefer to have WebRTC disabled unless a dedicated new PACKAGECONFIG option is enabled. Then you could make it depend on libvpx too.

It is already an option [L47](https://github.com/Igalia/meta-webkit/pull/298/files#diff-59ec6e83a596e9b563488fbbae9abd8aba9f350d4b8e9de9d5cb38b6259a81eaL47) and not selected by default  [L20](https://github.com/Igalia/meta-webkit/pull/298/files#diff-59ec6e83a596e9b563488fbbae9abd8aba9f350d4b8e9de9d5cb38b6259a81eaL20)

File: recipes-browser/wpewebkit/wpewebkit.inc [hunk 1]
```diff
@@ -51,7 +52,7 @@ PACKAGECONFIG[unified-builds] = "-DENABLE_UNIFIED_BUILDS=ON,-DENABLE_UNIFIED_BUI
 PACKAGECONFIG[systemd] = "-DUSE_SYSTEMD=ON,-DUSE_SYSTEMD=OFF,systemd"
 PACKAGECONFIG[thunder] = "-DENABLE_THUNDER=ON,-DENABLE_THUNDER=OFF,virtual/open-cdm"
 PACKAGECONFIG[video-plane-display-dmabuf] = "-DUSE_WPE_VIDEO_PLANE_DISPLAY_DMABUF=ON,-DUSE_WPE_VIDEO_PLANE_DISPLAY_DMABUF=OFF,wpebackend-fdo"
-PACKAGECONFIG[webxr] = "-DENABLE_WEBXR=ON,-DENABLE_WEBXR=OFF,"
+PACKAGECONFIG[webxr] = "-DENABLE_WEBXR=ON,-DENABLE_WEBXR=OFF,openxr"
 PACKAGECONFIG[jit] = "-DENABLE_JIT=ON,-DENABLE_JIT=OFF -DENABLE_C_LOOP=ON,"
 PACKAGECONFIG[gold] = "-DUSE_LD_GOLD=ON,-DUSE_LD_GOLD=OFF,"
 #TODO: Add recipe for openxr:
```
Comment (philn) [inline recipes-browser/wpewebkit/wpewebkit.inc:58]: You can remove this TODO :)

Comment (psaavedra) [reply recipes-browser/wpewebkit/wpewebkit.inc:58]: ouch!

Comment (psaavedra) [reply recipes-browser/wpewebkit/wpewebkit.inc]: Done

**Review by philn — APPROVED** [review]

**Pull request merged**